docs/developers/nft-tokens.md

```diff
@@ -354,13 +358,21 @@ Additional gas refers to:
 - Transaction payload cost: Data field length * 1500 (GasPerDataByte = 1500)
 - Storage cost: Size of NFT data * 50000 (StorePerByte = 50000)
 
+To see see more about the fields that are needed, please refer to [this](/developers/nft-tokens#nftsft-fields) section.
+
 :::tip
 Note that because NFTs are stored in accounts trie, every transaction involving the NFT will require a gas limit depending on NFT data size.
 :::
 
+Most of the times you will be able to create the NFTs by issuing one single transaction.
+This assumes the metadata file as well as the NFT media is already uploaded to IPFS.
+
+There are times however when uploading the metadata file before issuing the NFT is not possible (eg. when issued from a smart contract)
+In this cases it is possible to update an NFT with the metadata file after it was issued by sending an additional transaction [here](/developers/nft-tokens#change-nft-attributes) you can find more information about how to update the attributes
```

Comment:
by sending an additional transaction.
You can find more information [here] ... about how...